.github/workflows/ci.yml

```diff
@@ -27,6 +27,11 @@ jobs:
             pytest-md
             pytest-emoji
 
+      - name: Install quetz
+        run: |
+          micromamba install -c conda-forge quetz
+          micromamba install -c conda-forge sqlalchemy=1.4.46
```

Comment:
We can probably remove the pin now.

Reply:
[Not quite yet](https://github.com/mamba-org/quetz-client/actions/runs/4184517170/jobs/7250270553), but soon it will work 😅